api/v1alpha1/machine_types.go

```diff
@@ -0,0 +1,64 @@
+//  SPDX-FileCopyrightText: Alice Frosi <afrosi@redhat.com>
```

Comment:
nit: Any reason why separating the Machine in a second file?

Reply:
The Kubebuilder quickstart suggests this but indeed they can go to one file.